DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
+    "title": "Symantec DLP",
+    "publisher": "Broadcom",
+    "descriptionMarkdown": "The [Broadcom Symantec Data Loss Prevention (DLP)](https://www.broadcom.com/products/cyber-security/information-protection/data-loss-prevention) connector allows you to easily connect your Symantec DLP with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization’s information, where it travels, and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **SymantecDLP** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-symantecdlp-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "Symantec DLP",
+        "baseQuery": "CommonSecurityLog​ \n| where DeviceVendor == \"Symantec\" and DeviceProduct == \"DLP\""
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Triggered Activities",
+        "query": "SymantecDLP \n | summarize count() by Activity \n| top 10 by count_"
+    }, {
+        "description": "Top 10 Filenames",
+        "query": "SymantecDLP \n | summarize count() by FileName \n| top 10 by count_"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": ["CommonSecurityLog​ \n| where DeviceVendor == \"Symantec\" and DeviceProduct == \"DLP\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"]
```

Comment:
Please remove [ before CommonSecurityLog and ] at the end, they are not required.

Reply:
changes have been pushed. This is taken from the template from the Palo Alto connector page.